clever.wdl

```diff
@@ -0,0 +1,66 @@
+version 1.0 
+
+import "common.wdl"
+
+task Prediction {
+    input {
+        IndexedBamFile bamFile
+        Reference reference
+        String outputPath        
+        Int threads = 10 
+    }   
+    
+
+    command <<< 
+        clever \
+        -T ~{threads} \
+        --use_mapq \
+        --sorted \
+        -f ~{bamFile.file} \
+        ~{reference.fasta} \
+        ~{outputPath}
+    >>> 
+
+    output {
+        File predictions = "~{outputPath}/predictions.vcf"
+    }   
+    
+    runtime {
+        cpu: threads
+        docker: "quay.io/biocontainers/clever-toolkit:2.4--py37hcfe0e84_5"
+    }   
+
+}
+
+task Mateclever {
+    input {
+        IndexedBamFile bamFile
+        Reference reference
+        File predictions
+        String outputPath
+        Int threads = 10 
+    }
+
+    command <<<
+        echo ~{outputPath} ~{bamFile.file} ~{predictions} none > ~{outputPath}.list
```

Comment:
since there are multiple bash commands in this command section, please add `set -e` at the top